treeherder/client/thclient/client.py

```diff
@@ -752,6 +753,16 @@ def get_jobs(self, project, **params):
         response = self._get_json(project, self.JOBS_ENDPOINT, None, **params)
         return response["results"]
 
+    def get_artifact(self, project, **params):
+        """
+        Gets an artifact (a python list) from project, filtered by parameters
```

Comment:
To be consistent with the other API methods, let's just say "Gets artifact list from project, filtered by parameters"